backend/Squirrel.ConsoleApp/Extensions/HubConnectionExtensions.cs

```diff
@@ -102,5 +103,11 @@ public static void RegisterActions(this HubConnection hubConnection, IApplicatio
             var getActionsService = app.ApplicationServices.GetRequiredService<IGetActionsService>();
             hubConnection.InvokeAsync("ProcessReceivedDataFromClientSide", queryId, "GetUserDefinedTableTypesAsync", getActionsService.GetUserDefinedTableTypesAsync().Result);
         });
+        
+        hubConnection.On("RemoteConnectAsync", (Guid queryId, ConnectionStringDto connectionStringDto) =>
```

Comment:
I hope we will move all this (`"RemoteConnectAsync"`) magic strings to common, but in some next PR